cmd/geteduroam/main.go

```diff
@@ -208,19 +209,20 @@ func askPassword() string {
 // askCredentials asks the user for credentials
 // It returns the username and password
 func askCredentials(c network.Credentials, pi network.ProviderInfo) (string, string) {
-	fmt.Println("\nOrganization info:")
-	fmt.Println(" Title:", pi.Name)
-	fmt.Println(" Description:", pi.Description)
-	if pi.Helpdesk.Email != "" {
-		fmt.Println(" Helpdesk e-mail:", pi.Helpdesk.Email)
-	}
-	if pi.Helpdesk.Phone != "" {
-		fmt.Println(" Helpdesk phone number:", pi.Helpdesk.Phone)
-	}
-	if pi.Helpdesk.Web != "" {
-		fmt.Println(" Helpdesk URL:", pi.Helpdesk.Web)
+	if utils.IsVerbose {
```

Comment:
I think this should always be shown as it's inherit to the UI

Reply:
I thought this was a nice example of where verbose could prove it's worthiness?

Reply:
Well I think it's nice to always show these messages as the GUIs also always show it. What we're currently missing anyways is terms of use which should always be shown and is closely related to this information. I think verbose could be great for actually giving you an idea what the tool is doing, e.g.

getting list of organizations...

requesting eap file with url: ....

parsing eap file...

done parsing eap file...

configuring connection...